datasets/hst.yaml

```diff
@@ -0,0 +1,26 @@
+Name: HST public data
+Description: |
+  The Hubble Space Telescope (HST) is one of the most productive scientific instruments ever created. This dataset contains calibrated and raw data for all of the currently active instruments on HST: ACS, COS, STIS and WFC3. 
+Documentation: http://astroquery.readthedocs.io/en/latest/mast/mast.html
+Contact: archive@stsci.edu
+UpdateFrequency: Hourly
+Tags:
+  - astronomy
+  - hst
+  - hubble
+  - nasa
+License: STScI herby grants the non-exclusive, royalty free, non-transferable, worldwide right and license to use, reproduce and publicly display in all media public data from the Hubble Space Telescope.
+Resources:
+  - Description: Project data files
+    ARN: arn:aws:s3:::stpubdata
+    Region: us-east-1
+    Type: S3 Bucket
+  - Description: Notifications for new data
+    ARN: arn:aws:sns:us-east-1:928094251383:gdelt-csv
+    Region: us-east-1
+    Type: SNS Topic
+DataAtWork:
+  - Title: Making HST Public Data Available on AWS
+    URL: http://mast-labs.stsci.io
```

Comment:
This isn't public yet. Also, I think we might want to cross-post on the MAST blog as well as the AWS blog. Will follow up with email to @jedsundwall